app/src/main/res/values/strings.xml

```diff
@@ -1,8 +1,7 @@
 <resources>
-    <string name="app_name">Complexify-Android</string>
+    <string name="app_name">Complexify sample app</string>
 
-    <string name="hello_world">Hello world!</string>
-    <string name="action_settings">Settings</string>
+    <string name="minimum_complexity">0.0</string>
```

Comment:
Can't this be read from a Complexify configuration or is it defined per call site?